kiwi/tasks/image_info.py

```diff
@@ -101,7 +101,17 @@ def process(self):
             solver = self._setup_solver()
             package_list = self.xml_state.get_bootstrap_packages() + \
                 self.xml_state.get_system_packages()
-            solved_packages = solver.solve(package_list)
+            bootstrap_packages = solver.solve(
+                self.xml_state.get_bootstrap_packages(), False,
+                True if self.xml_state.get_bootstrap_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages = solver.solve(
+                self.xml_state.get_system_packages(), False,
+                True if self.xml_state.get_system_collection_type() is
+                'onlyRequired' else False
+            )
+            solved_packages.update(bootstrap_packages)
```

Comment:
I guess you moved from one solver operation to two solver operations because in theory we allow two different collection types between bootstrap and image packages processing, right ?

The only point I have here is that this will slow down the operation a bit. As there is no way out I just would like to ask if you have tested a real solver operation and have checked if the results stayed the same compared to the former code ?

Thanks